frontend/util.lua

```diff
@@ -80,8 +80,9 @@ function util.secondsToClock(seconds, withoutSeconds)
             return "00:00:00";
         end
     else
+        local round = withoutSeconds and require("optmath").round or math.floor
```

Comment:
@robert00s Any thoughts on this change btw? It's used a bunch in Statistics (just for display afaik). I thought it was kind of odd to always floor.

00:01:59 → 00:02 (really?)
now:
00:01:29 → 00:01
00:01:30 → 00:02

Reply:
This change look good for me :+1:

Reply:
Alright, thanks. :-)